parts/linux/cloud-init/artifacts/cse_install.sh

```diff
@@ -147,6 +147,31 @@ installCredentalProvider() {
     rm -rf ${CREDENTIAL_PROVIDER_DOWNLOAD_DIR}
 }
 
+# TODO (alburgess) have oras version managed by dependant or Renovate
+installOras() {
```

Comment:
Suggested to add unit test with shellspec. Please see the instructions in the readme.